### PR TITLE
Jdk11 compilable

### PR DESCRIPTION
**Pull request by je-ik**

### CODE REVIEW

File: maven/pom.xml [hunk 1]
```diff
@@ -31,12 +31,16 @@
      
   <name>${project.groupId}:${project.artifactId}</name>
 
+  <properties>
+    <maven.plugin.version>3.6.1</maven.plugin.version>
```
Comment (LesTR) [inline maven/pom.xml]: I don't see any usage of this propertie, is it correct?

File: pom.xml [hunk 2]
```diff
@@ -220,17 +220,46 @@
     </extensions>
 
     <plugins>
-      <plugin>
+      <!--plugin>
```
Comment (LesTR) [inline pom.xml]: please remove this commented block

File: maven/pom.xml (hunk 1 above)
Comment (je-ik) [reply maven/pom.xml]: I planned to use it, but it turned out, that all maven dependencies have different versions. Will remove it.

File: pom.xml (hunk 2 above)
Comment (je-ik) [reply pom.xml]: :+1:

**Pull request merged**